I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 1412, in script
    show happy_lolajiji at right, lughten with dissolve
  File "game/script.rpy", line 1412, in <module>
    show happy_lolajiji at right, lughten with dissolve
NameError: name 'lughten' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 1412, in script
    show happy_lolajiji at right, lughten with dissolve
  File "/Applications/renpy-8.3.2-sdk/renpy/ast.py", line 1124, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "/Applications/renpy-8.3.2-sdk/renpy/ast.py", line 1083, in show_imspec
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "/Applications/renpy-8.3.2-sdk/renpy/ast.py", line 1083, in <listcomp>
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "/Applications/renpy-8.3.2-sdk/renpy/python.py", line 1215, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "/Applications/renpy-8.3.2-sdk/renpy/python.py", line 1208, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 1412, in <module>
    show happy_lolajiji at right, lughten with dissolve
NameError: name 'lughten' is not defined

macOS-15.3.1-arm64-arm-64bit arm64
Ren'Py 8.3.6.25022803
kinaiyagame_3.0 1.0
Wed Mar 12 00:10:23 2025
